libs/server/ACL/ACLPassword.cs

```diff
@@ -92,20 +92,7 @@ public override string ToString()
         /// <returns>True if equal, otherwise false.</returns>
         public bool Equals(ACLPassword password)
         {
-            if (PasswordHash.Length != password.PasswordHash.Length)
-            {
-                return false;
-            }
-
-            for (int i = 0; i < PasswordHash.Length; i++)
-            {
-                if (PasswordHash[i] != password.PasswordHash[i])
-                {
-                    return false;
-                }
-            }
-
-            return true;
+            return SecretsUtility.ConstantEquals(password.PasswordHash.AsSpan(), PasswordHash.AsSpan());
         }
```

Comment:
Is there a reason not to use `CryptographicOperations.FixedTimeEquals`?

Reply:
I couldn't remember the name of the method when I searched for it. 😅 I'll update that.